### PR TITLE
Update List.cs

### PR DESCRIPTION
**Pull request by viveknuna**
The comment was misleading, It should be removing the item not the item from given index

### CODE REVIEW

File: src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs [hunk 1]
```diff
@@ -866,7 +866,7 @@ public int LastIndexOf(T item, int index, int count)
             return Array.LastIndexOf(_items, item, index, count);
         }
 
-        // Removes the element at the given index. The size of the list is
+        // Removes the given element. The size of the list is
```
Comment (eiriktsarpalis) [inline src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs:869]:
```suggestion
        // Removes the first occurrence given element, if found. The size of the list is
```

Comment (viveknuna) [reply src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs:869]: Yes, You are right @eiriktsarpalis

Comment (viveknuna) [reply src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs]: @eiriktsarpalis Thank you for reviewing, I have made the changes, Please review now.

**Review by eiriktsarpalis — APPROVED** [review]
Thanks!

File: src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs [hunk 2]
```diff
@@ -866,7 +866,7 @@ public int LastIndexOf(T item, int index, int count)
             return Array.LastIndexOf(_items, item, index, count);
         }
 
-        // Removes the element at the given index. The size of the list is
+        // Removes the first occurrence given element, if found. The size of the list is
```
Comment (danmoseley) [inline src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs:869]:
```suggestion
        // Removes the first occurrence of the given element, if found. The size of the list is
```

Comment (eiriktsarpalis) [reply src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs:869]: Ugh. I'll follow up with another PR.

**Pull request merged**